python/ray/air/_internal/torch_utils.py

```diff
@@ -1,71 +1,31 @@
-import os
 import warnings
 from typing import Dict, List, Optional, Union, Any
 
 import numpy as np
 import pandas as pd
-import torch
 
-import ray
+from ray.air._internal.accelerator_utils import try_import_torch
 from ray.air.util.data_batch_conversion import _unwrap_ndarray_object_type_if_needed
 
-from ray._private.accelerators.hpu import HPU_PACKAGE_AVAILABLE
 
-if HPU_PACKAGE_AVAILABLE:
-    import habana_frameworks.torch.hpu as torch_hpu
+torch, _ = try_import_torch()
```

Comment:
`torch_utils.py` still need import torch since it provides public functions related to torch. However, we have already separated the torch and torch_xpu based on [this suggestion](https://github.com/ray-project/ray/pull/44086#discussion_r1539924222).